src/Control/Effect/Labelled.hs

```diff
@@ -46,7 +47,7 @@ import Data.Kind (Type)
 --
 -- @since 1.0.2.0
 newtype Labelled (label :: k) (sub :: (Type -> Type) -> (Type -> Type)) m a = Labelled (sub m a)
-  deriving (Alternative, Applicative, Functor, Monad, Fail.MonadFail, MonadIO, MonadPlus, MonadTrans)
+  deriving (Alternative, Applicative, Functor, Monad, Fail.MonadFail, MonadIO, MonadPlus, MonadFix, MonadTrans)
```

Comment:
Could you please keep this alphabetized (by class name)? i.e. insert `MonadFix` between `Fail.MonadFail` and `MonadIO`.